### ISSUE TITLE
Magnifier does not follow text insertion point in VS Code

### ISSUE THREAD

**Issue by coryj627** [microsoft/vscode#77842]
Accessibility issue. Breaks [WCAG 4.1](https://www.w3.org/TR/WCAG21/#compatible)

- VSCode 1.36.1
- Win 10 Pro 1903
- 3840 x 2160 resolution display
- Visual Studio 2019 16.1.6
- Edge Dev 77.0.223.0 
- Slack 4.0.0
- Windows Magnifier

Steps to Reproduce:

1. Open the start menu and navigate to settings
2. Open "Ease of Access"
3. Open the "Magnifier" panel from the sidebar
4. Under "Change Magnifier view" set "Choose a view" to "Full Screen"
5. Under "Have Magnifier follow:" check the box to enable "Text insertion point"
6. Enable all of the other items under "Have Magnifier follow"
7. Close the Settings window
8. Open VS Code Windows
9. Open a long file or create a new blank file.
10. Press Windows Key and ' + ' to enable Windows Magnifier
11. Press Windows key and ' + '  again until the view is magnified so that you can only see a portion of the text editor canvas
12. Attempt to type a few sentences with many line breaks in between.
13. Observe that the Magnifier does not follow the insertion point as it does in other apps
14. Use the left, right, up, down arrow keys to move the insertion point around the text and observe that the view-port does not follow
15. Place the insertion point at the end of the document by pressing ctrl + end
16. Use your mouse to focus on this area of the editor
17. Attempt to start adding text - observe the focus shifts to the upper left of the screen.

Repeat this experiment with the Monaco editor playground on Edge Dev or Chrome to see that it is not Monaco, repeat it again with another Electron app optimized for accessibility (I used Slack 4.0.0) and see that it is not an issue of the stable release of Electron.

Here is a [video reviewing these experiments](https://1drv.ms/v/s!Ar24c3vrvBZxoMsYKOXcMoI_uPNmeQ?e=eYJtVX).

Does this issue occur when all extensions are disabled?: Yes

This apears to be an ongoing issue that has been repeatedly closed... What's the story here?
#43639 
#35613 
#35396 
#33976
#29253 
#7394

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/7394
Please find the details there and we can continue the discussion in that issue. Thank you

**Comment by isidorn** [comment]
Please note the chrome issue https://bugs.chromium.org/p/chromium/issues/detail?id=142133&can=2&start=0&num=100&q=&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified&groupby=&sort=
So it seems like the issue might have regressed.
Also note that we use electron `4.2.5`

Commenting on the chrome issue would be very benefical also.
